.eslintignore

```diff
@@ -1,3 +1,4 @@
 **/node_modules/*
 **/dist/*
-.github/actions/**/index.js"
+.github/actions/**/index.js
+docs/vendor/*
```

Comment:
Should this be pushed?

Reply:
Yes, we want to ignore `docs/vendor/*` in eslint. Not a problem on CI because that directory won't exist on runners. But if you build the helpsite locally then run eslint you'll get a lot of false flag failures.

I'll move this to a separate PR